src/style.css

```
@@ -1 +1,67 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+.body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.home__link {
+  margin: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav__list {
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+  position: relative;
+}
```

```suggestion
.nav__link {
  text-decoration: none;
  text-transform: uppercase;
  color: #000;
  padding: 23px 0;
  display: inline-block;
}
```
add display inline block and run tests again